templates/answerfile_4.2_basic.txt.j2

```diff
@@ -0,0 +1,9 @@
+{% include "./templates/basic_answerfile.txt.j2" %},
+[environment:default]
+OVESETUP_OVN/ovirtProviderOvn=bool:True
```

Comment:
This should be provided as role parameter

Reply:
@mwperina  oVrt.repositories is not really a dependency, since if the target machines has the repositories by some other way (user brought them manually) role will work fine.